docs/documentation/release_notes/topics/26_3_0.adoc

```diff
@@ -29,3 +29,12 @@ details, see link:{upgradingguide_link}[{upgradingguide_name}].
 = Recovery Codes supported
 
 In this release, the *Recovery Codes* two-factor authentication is promoted from preview to supported feature. For more information about the 2FA method, see the link:{adminguide_link}#_recovery-codes[Recovery Codes] chapter in the {adminguide_name}.
+
+= Experimental support for rolling updates for patch releases
+
+In the previous release, we added support into Keycloak Operator to perform rolling update of the Keycloak image if both images contain the same version.
+This is useful, for example, when switching to an optimized image, changing a theme or a provider source code.
```

Comment:
"In the previous release, the Keycloak Operator was enhanced to support performing rolling updates of the Keycloak image provided that both images contain the same version."

Slight change to the grammar. Feel free to ignore :slightly_smiling_face:

Reply:
done